src/db/tables/breaches.js

```diff
@@ -18,6 +18,20 @@ async function getAllBreaches() {
     .returning("*")
 }
 
+/**
+ * Get all count from "breaches" table
+ *
+ * @returns Count of all records from "breaches" table
+ */
+async function getAllBreachesCount() {
+  const breachesCount = await knex('breaches')
+    .count({ count: "*" })
+    .first()
+    .then(result => result?.count)
+  // @ts-ignore The above query is returning a string
+  return parseInt(breachesCount, 10)
```

Comment:
```suggestion
  return parseInt(breachesCount?.toString() || "", 10)
```

Reply:
Thanks — changed in [bd3a6df](https://github.com/mozilla/blurts-server/pull/3233/commits/bd3a6df8c2ed1e6c2e59780192ad8b0f2fc822d6).